csrankings-c.csv

```diff
@@ -684,7 +684,7 @@ Christian Guetl,Graz University of Technology,https://www.tugraz.at/institutes/i
 Christian Gütl,Graz University of Technology,https://www.tugraz.at/institutes/isds/home,NoAiKpcAAAAJ
 Christian Hammer 0001,University of Potsdam,https://www.uni-potsdam.de/en/cs-se/team/prof-dr-ing-christian-hammer,gykAfPwAAAAJ
 Christian Hansen,University of Magdeburg,http://isgwww.cs.uni-magdeburg.de/cas,qV62eOQAAAAJ
-Christian Hardmeier,IT University of Copenhagen,https://pure.itu.dk/portal/en/persons/christian-hardmeier(9504265d-0046-4e58-b529-256a656c012d).html,AmRp1T0AAAAJ
+Christian Hardmeier,IT University of Copenhagen,https://christianhardmeier.rax.ch/,AmRp1T0AAAAJ

```

Comment:
This person is currently away from their institution and needs to be removed (" I’m currently staying in Edinburgh, where I’ve spent 1.5 years as a visitor and Senior Researcher at the School of Informatics of the University of Edinburgh. ") - sabbatical leaves are fine but this is > 1 year.

Reply:
Christian moved from Edinburgh to ITU, hence this is an incorrect leftover on this webpage. I revert back to the pure homepage instead of his personal page, which can also be found through Google scholar

Reply:
Update: I notified him about this leftover on his website, thanks for spotting it.